src/middlewared/middlewared/plugins/kubernetes_linux/app_migrations.py

```diff
@@ -143,7 +165,21 @@ def load_migrations(self, catalog):
 
             return migrations
         else:
-            return {}
+            return migrations
+
+    def official_migrations(self):
+        migrations = {}
```

Comment:
This can be simplified and made more performant (there is no reason for sorting)
```suggestion
        migrations = {}
        app_migration_dir = f'{get_middleware_dir()}/plugins/kubernetes_linux/app_migrations'
        for migration in filter(lambda x: x.suffix == '.json', Path(app_migration_dir).iterdir():
            migrations[migration.name] = json.loads(migration.read_text())
        return migrations
```